src/components/ProvinceDropdown/province.data.js

```diff
@@ -0,0 +1,392 @@
+export default [
+  {
+    code: "TH-10",
+    title_th: "กรุงเทพมหานคร",
+    type: "เขตปกครองพิเศษ",
+  },
+  {
+    code: "TH-11",
+    title_th: "สมุทรปราการ",
+    type: "จังหวัด",
```

Comment:
Also, I'm not entirely sure where `type` will be used, but if it'll be sent to the server, you might want to use the English key names instead.

Of course, if it's just used for displaying on the frontend, then no adjustments needed here.

Reply:
I didn't discuss the province data with our team, so I want to leave it until our team has a final decision.